README.md

```diff
@@ -1,3 +1,161 @@
-# Run Parasoft SOAtest action
+# Run Parasoft SOAtest
 
-This action enables you to run functional tests with Parasoft SOAtest and review results on GitHub.
\ No newline at end of file
+[![Build](https://github.com/parasoft/run-soatest-action/actions/workflows/build.yml/badge.svg?branch=master)](https://github.com/parasoft/run-soatest-action/actions/workflows/build.yml)
+
+This action enables you to run static analysis and execute tests with Parasoft SOAtest and review results on GitHub.
+
+Parasoft SOAtest is an enterprise-grade solution that simplifies complex testing for business-critical transactions through APIs, message brokers, databases mainframes, ERPs, browser-based UIs, and other endpoints. SOAtest helps QA teams ensure secure, reliable, compliant business applications with an intuitive interface to create, maintain, and execute end-to-end testing scenarios.
```

Comment:
Should refer to [https://gitlab.com/parasoft/soatest-gitlab](https://gitlab.com/parasoft/soatest-gitlab) for SOAtest description and the bullet points.